playground/src/import-url/import-url.js

```diff
@@ -89,13 +87,13 @@ class ImportURL extends FlyIn {
         this.importError(e);
       })
       .finally(() => {
-        this.urlBarSubmit.classList.remove('loading');
+        this.inputBar.toggleLoading();
```

Comment:
Hm... this might be safe as the submit button is guarded against double submits - but in a worst case scenario this could start loading again.